go-engine/unleash_engine_test.go

```diff
@@ -212,6 +213,10 @@ func TestClientSpecification(t *testing.T) {
 
 				result := engine.GetVariant(toggleName, context)
 
+				if result == nil {
+					result = &VariantDef{"disabled", nil, false, engine.IsEnabled("Feature.A", context)}
```

Comment:
I think this should just be false. Tests pass (on my machine!) with this change but if there's a reason we need to do the evaluation for the actual tests, then probably we should do something like check the actual test toggle that we're testing

```suggestion
					result = &VariantDef{"disabled", nil, false, false}
```